src/tools/wasm-split/wasm-split.cpp

```diff
@@ -385,6 +398,53 @@ void mergeProfiles(const WasmSplitOptions& options) {
   buffer.writeTo(out.getStream());
 }
 
+// todo: replace with better way converting to readable fn name
+std::string replaceHexNumbersWithChar(std::string input) {
+  std::string output;
+  for (size_t i = 0; i < input.length(); i++) {
+    if ((input[i] == '\\') && (i + 2 < input.length())) {
+      std::string byte = input.substr(i + 1, 2);
+      i += 2;
+      char chr = (char)(int)strtol(byte.c_str(), nullptr, 16);
+      output.push_back(chr);
+    } else {
+      char chr = input[i];
+      output.push_back(chr);
+    }
+  }
+  return output;
+}
+
+void printReadableProfile(const WasmSplitOptions& options) {
+  Module wasm;
+  parseInput(wasm, options);
+
+  std::set<Name> keepFuncs;
+  std::set<Name> splitFuncs;
+
+  if (options.profileFile.size()) {
```

Comment:
Should we error out if there is no profile supplied?

Reply:
yes, I do see an early check helps exit quickly if profile does not exist. 
done